src/constants.ts

```diff
@@ -7,7 +7,7 @@ export const PROGRAM_VERSION = packageDotJson.version;
 export const USER_AGENT = `CrossSeed/${PROGRAM_VERSION}`;
 
 export const EP_REGEX =
-	/^(?<title>.+?)[\s._](?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)/i;
+	/^(?<title>.+?)[\s._](?>(?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)|(?<date>\d{4}.\d{2}.\d{2}))/i;
 export const SEASON_REGEX =
 	/^(?<title>.+?)[_.\s](?<season>S\d+)(?:[.\-\s]*?(?<seasonmax>S?\d+))?(?=[_.\s](?!E\d+))/i;
```

Comment:
The same applies with a positive lookahead here. This was an oversight on my part before.

Also fixed a failure to capture seasonmax due to missing the underscore.

```suggestion
	/^(?<title>.+?)[_.\s](?=(?<season>S\d+)(?:[.\-_\s]*?(?<seasonmax>S?\d+))?(?=[_.\s](?!E\d+)))/i;
```

Test cases -> https://regex101.com/r/96yjzz/2